### ISSUE TITLE
Snowfall S01 E01 matching to a release named "S.S01E01..."

### ISSUE THREAD

**Issue by RaboonTheBaboon** [Sonarr/Sonarr#8897]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

Add the series "Snowfall" with 1080p quality
Manual search for Season 1 Episode 1
The top release that shows, and is grabbed is

S.S01E01.PROPER.1080p.NOW.WEB-DL.DDP5.1.Atmos.H.264-JFF
Which is tagged as a proper.

### Expected Behavior

"Snowfall" should not match an upload titled "S".

Thank you!

### Steps To Reproduce

As above

### Environment

```markdown
- OS: Unraid 7.3.2
- Sonarr: 4.0.19.3001
- Docker Install: Yes, linuxserver
- Using Reverse Proxy: No
- Browser: Safari 26.6.2 (21624.5.1.11.3)
- Database: Sqlite 3.53.4
```

### What branch are you running?

Develop

### Trace Logs

[sonarr.trace.txt](https://github.com/user-attachments/files/31334779/sonarr.trace.txt)

### Anything else

<img width="1470" height="457" alt="Image" src="https://github.com/user-attachments/assets/62ac33c1-00ae-45da-81a2-b508dd59f60d" />

**Comment by PearsonFlyer** [comment]
This happened because a search was made using TVDBID, and your indexer improperly matched it. You should report this mismatch to your indexer, but it's not Sonarr's fault, and it would have been stopped from importing. This is a support issue, and you should use reddit, the forums, or discord to resolve it, and not raise a github issue.

**Comment by RaboonTheBaboon** [comment]
Thanks for the guidance @PearsonFlyer
